### PR TITLE
[new] add audiojs 1.0.1

### PR DESCRIPTION
**Pull request by EndenDragon**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kolber/audiojs
 * Official website (optional, not the repository): http://kolber.github.io/audiojs/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1458
   - Count of watchers: 86
   - Count of forks: 331
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@EndenDragon would you please also minify audio.cmd.js, and reword the commit message to something like `add audio.js`? Thanks.

File: ajax/libs/audiojs/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "author": "Anthony Kolber",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kolber/audiojs.git",
+    "basePath": "",
+    "files": [
+      "audiojs/audio.js",
+      "audiojs/audio.min.js",
+      "audiojs/audiojs.swf",
+      "audiojs/audio.cmd.js",
```
Comment (PeterDaveHello) [inline ajax/libs/audiojs/package.json]: `"audiojs/audio*.+(js|swf)"` will be much more simplier.

**Comment by EndenDragon** [comment]
How would I change the commit message? Also, how can I minify the code? By minfying, do I just add a min file to ajax/libs/audiojs/1.0.1 folder?

**Comment by PeterDaveHello** [comment]
You missed the auto-update filemap for the gif file in the new commit.

**Pull request merged**